src/components/ui/logo-skeleton.tsx

```diff
@@ -0,0 +1,9 @@
+import { cn } from "@/lib/utils";
+
+const Skeleton = () => {
+  return (
+    <div className={cn("animate-pulse bg-muted h-[60px] w-[60px] rounded-md")}></div>
```

Comment:
This is a reusable component so we can't define height and width here directly. Use this code to make it reusable and provide height and width where this skeleton is being used

```tsx
import { cn } from "@/lib/utils"
 
function Skeleton({
  className,
  ...props
}: React.HTMLAttributes<HTMLDivElement>) {
  return (
    <div
      className={cn("animate-pulse rounded-md bg-muted", className)}
      {...props}
    />
  )
}
 
export { Skeleton }
```

Reply:
Also, rename file to `skeleton.tsx` instead of `logo-skeleton` because we'll be using the same skeleton component everywhere